projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -70,8 +88,20 @@ export class TabParagraphContainerComponent
     )
   );
 
-  select(tabNum: number): void {
-    this.activeTabNum = tabNum;
+  select(tabNum: number, event?: Event): void {
+    this.breakpointService
+      .isDown(BREAKPOINT.sm)
+      .pipe(take(1))
+      .subscribe((res) => {
+        if (res) {
+          this.activeTabNum = this.activeTabNum === tabNum ? -1 : tabNum;
+          if (event && event['path']) {
```

Comment:
1. We should not use property `path` of the event, becuase it's not supported i.e. by Firefox. The properties that are safe to use should exist in the type `MouseEvent`. Using proper types helps avoiding such bugs.

2. Btw. For type safety we should not access properties by `event['propertyName']` (which is a workaround to cheat the typing), but `event.propertyName`.

Reply:
Hey @Platonn - I checked event?.composedPath() as its supported across all browsers except IE 10.. https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath

do you have any other recommendation to use?

Reply:
event.target as HTMLElement tested across edge, firefox & chrome & it works